users/templates/my_account.html

```diff
@@ -64,25 +64,12 @@ <h1>Edit your account</h1>
                     <form method="POST" enctype="multipart/form-data">
                         {% csrf_token %}
                         {#{{ user_form.as_p }}#}
-                        {#{{ profile_form.as_p }}#}
+                        {% for field in user_form %}
```

Comment:
@SahinKapan We shouldn't type in the form fields statically, they should be dynamic and can be controlled from the backend without needing to changing html files. Can you please refactor the forms like this? I can help you with that if you need.